.github/workflows/node-ci.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-latest
     services:
       rabbitmq:
-        image: rabbitmq
+        image: rabbitmq:3.9-alpine
```

Comment:
alpine should be faster to download. I also noticed some odd problems with the ubuntu version where it looked like ubuntu wasn't able to install rabbitmq.

Reply:
Any reason why you are not using the latest version?

Reply:
Good spot, I'll change to 3-alpine.